playbook/app/pb_kits/playbook/pb_typeahead/docs/_typeahead_with_pills_async.html.erb

```diff
@@ -27,4 +36,11 @@
       }
     })
   }
+
+  window.getOptionLabel = function(option) {
+    return option.name;
+  }
+  window.getOptionValue = function(option) {
+    return option.id;
+  }
```

Comment:
From the Rails plain JS side, here is how you provide the function each prop is expecting.